pkg/apis/picchu/v1alpha1/releasemanager_types.go

```diff
@@ -52,7 +52,10 @@ type ReleaseManagerRevisionStatus struct {
 	LastUpdated          *metav1.Time                        `json:"lastUpdated"`
 	GitTimestamp         *metav1.Time                        `json:"gitTimestamp,omitempty"`
 	RevisionTimestamp    *metav1.Time                        `json:"revisionTimestamp,omitempty"`
+	IncarnationCreated   *metav1.Time                        `json:"incarnationCreationTimestamp,omitempty"`
 	CanaryStartTimestamp *metav1.Time                        `json:"canaryStartTimestamp,omitempty"`
+	IncarnationDeployed  *metav1.Time                        `json:"incarnationDeployedTimestamp,omitempty"`
+	IncarnationReleased  *metav1.Time                        `json:"incarnationReleasedTimestamp,omitempty"`
```

Comment:
Why do we care about incarnation times? Isn't this leaking the impl? If this is purely for debugging, it's fine, but I don't think it should be an SLI.

Reply:
basically im trying to measure time to initial ramp from when picchu knows about a revision to the time customers are using that revision. I'll rename it `DeployedTimestamp` and `ReleasedTimestamp`

Reply:
I think revision's creationTimestamp will be more accurate